snapcraft/common.py

```diff
@@ -98,20 +103,36 @@ def __init__(self):
         super().__init__(
             '{0} is not supported, please log a bug at '
             'https://bugs.launchpad.net/snapcraft/+filebug?'
-            'field.title=please+add+support+for+{0}'.format(
-                platform.machine()))
+            'field.title=please+add+support+for+{0}'.format(_machine))
```

Comment:
This is stable. If it comes to it, it would need to be changed like any other link we have.